app/WCSIM_TreeConvert.cc

```diff
@@ -747,14 +753,19 @@ int main(int argc, char **argv){
         nReflec = 0;
         nRaySct = 0;
         nMieSct = 0;
+        nReflec = HitTime->GetReflection();
```

Comment:
Just bear in mind that such counting is only correct for single Cherenkov hit per PMT.